### PR TITLE
Modified Payments Struct

### PR DESCRIPTION
**Pull request by tarunguguloth**
finalised Struct of payments

### CODE REVIEW

File: internal/model/payments.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import "time"
+
+type Payments struct {
+	Id             string    `gorm:"primary_key; column:id" json:"id"`
+	Userid         string    `gorm:"foreign_key; column:user_id" json:"user_id"`
+	RazorpayLinkId string    `gorm:"column:razorpay_link_id" json:"razorpay_link_id"`
+	RazorpayLink   string    `gorm:"column:razorpay_link" json:"razorpay_link"`
+	Amount         int       `gorm:"column:amount" json:"amount"`
+	Type           string    `gorm:"column:type" json:"type"`
```
Comment (mukesh0513) [inline internal/model/payments.go]: run gofmt.

File: internal/model/payments.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package Model
+
+import "time"
+
+type Payments struct {
+	Id             string    `gorm:"primary_key; column:id" json:"id"`
```
Comment (mukesh0513) [inline internal/model/payments.go]: why string?

File: internal/model/payments.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	UserId         string    `gorm:"foreign_key; column:user_id" json:"user_id"`
+	RazorpayLinkId string    `gorm:"column:razorpay_link_id" json:"razorpay_link_id"`
+	RazorpayLink   string    `gorm:"column:razorpay_link" json:"razorpay_link"`
+	Amount         int       `gorm:"column:amount" json:"amount"`
+	Type           string    `gorm:"column:type" json:"type"`
+	CurrentBalance int       `gorm:"column:current_amount" json:"current_amount"`
```
Comment (mukesh0513) [inline internal/model/payments.go]: is this current_amount or current_balance?

Comment (tarunguguloth) [reply internal/model/payments.go]: cahnges made please check

**Pull request merged**